src/plugins/intel_cpu/tests/unit/shape_inference_test/inverse_shape_inference_test.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "inverse_shape_inference.hpp"
+
+#include <gtest/gtest.h>
+
+#include "utils.hpp"
+
+using namespace ov;
+using namespace ov::intel_cpu;
+
+TEST(StaticShapeInferenceTest, InverseStaticShapeInferenceTest2D) {
```

Comment:
Add also test for default ctor but without validation check e.g. `MaxPool` test.
Just for information only, for these test also is helper fixture `OpStaticShapeInferenceTest` which also has `make_op` helper similar like in type_prop tests

Reply:
Added with `make_op`!